models/articles-models.js

```diff
@@ -21,22 +21,43 @@ exports.selectArticles = (sort_by = "created_at", order = "desc") => {
     "comment_count",
   ];
   const validOrder = ["asc", "desc"];
+  const validTopic = []
+  const queryValues = []
+  let topicString = `SELECT topic FROM articles`
```

Comment:
You declare this using a `let` but do not modify it. So is this a let, or a const?